src/app/app-routing.module.ts

```diff
@@ -12,6 +12,7 @@ import { SkillUsersReportComponent } from './reports/skill-users-report/skill-us
 import { SkillsComponent } from './skills/skills.component';
 import { UserProfileComponent } from './users/user-profile.component';
 import { SkillGroupsComponent } from './skill-groups/skill-groups.component';
+import { ProjectsComponent } from "./projects/projects.component";
```

Comment:
There should be quotes instead of double quotes.

Reply:
done